.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,34 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: "[BUG] "
+labels: bug
+assignees: ''
+
+---
+
+**Version**
+- Godwoken v1 or v0?
+- The prebuild-image you are using or the commit-sha1
```

Comment:
It is difficult for reporters to get the prebuild-images versions. We should teach them how to get this information. For example, `kicker info` or use `gw_get_node_info` RPC.

Reply:
https://github.com/nervosnetwork/godwoken-web3/pull/290/commits/d8660e2412682b35f52d6fb72efc1f5026877482